jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/cpu/OverallCPULoadHandler.java

```diff
@@ -50,16 +57,31 @@ public void initializeMeter(Meter meter) {
             measurement -> {
               measurement.record(machine);
             });
+    meter
+        .gaugeBuilder(METRIC_NAME_MACHINE_MINUTE)
+        .setDescription(METRIC_DESCRIPTION_MACHINE_MINUTE)
+        .setUnit(UNIT_UTILIZATION)
+        .buildWithCallback(
+            measurement -> {
+              measurement.record(machineMinuteAverage);
+            });
   }
 
   @Override
   public void accept(RecordedEvent ev) {
+    if (count % 60 == 0) {
```

Comment:
I think this 60 literal also needs to take into consideration the polling interval, right? So it needs to be computed.

Like if the interval changes to 2s, then the 60 would need to be changed to 30.

Reply:
We have a hardcoded `METRIC_NAME_MACHINE_MINUTE` and the metric defined in spec is, per, https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics specifically a 1-minute metric. So, I think this is OK - it isn't supposed to represent an average over a general interval.

Reply:
I think `process.runtime.jvm.system.cpu.load_1m` is intended to be a "load average", e.g. https://en.wikipedia.org/wiki/Load_(computing)#Unix-style_load_calculation and https://docs.oracle.com/javase/8/docs/api/java/lang/management/OperatingSystemMXBean.html#getSystemLoadAverage--, instead of an "average cpu utilization"

Reply:
ohh I see. I think that part of the semantic conventions was written with JMX in mind because JFR does not gather that information. A new event emitting that information will have to be created for this metric. Closing the PR.